apps/sample-app/src/components/Emails.tsx

```diff
@@ -0,0 +1,49 @@
+import { DocumentCard, DocumentCardTitle } from '@fluentui/react';
+import { Text } from '@fluentui/react-components';
+import { Message } from '@microsoft/microsoft-graph-types';
+import React from 'react';
+
+import { handleOpenMailItem } from './utils';
+
+interface EmailProps {
+  messages: Message[];
+}
+export interface MessageListItem {
+  key?: string;
+  subject?: string;
+  sender?: string;
+}
+export const EmailList: React.FC<EmailProps> = (props: EmailProps) => {
+  const { messages } = props;
+  const emailItems = messages.map<MessageListItem>(m => {
+    return {
+      key: m.id,
+      subject: m.subject || '',
+      sender: m.sender?.emailAddress?.name || '',
+    };
+  });
+
+  const EmailExample: React.FunctionComponent = () => {
+    return (
+      <div>
+        {emailItems.map(e => (
+          <Text as="span" key={e.key}>
```

Comment:
Could we rename `e` to be a bit longer? It looks like it's suppose to be an email item, maybe rename it to be `emailItem` or something like that? I do notice it sounds a lot similar to `emailItems` but I think it's still more explanatory than just the single character.

Reply:
Applied to all the other places in the PR where it references `e`

Reply:
Okay, also from an earlier PR I make a similar mapping function but only use 'c' (its in PeopleAvatars.tsx). I'll change those to be more descriptive as well